contributors/devel/sig-testing/testing.md

```diff
@@ -197,3 +197,61 @@ Please refer to [Integration Testing in Kubernetes](integration-tests.md).
 ## End-to-End tests
 
 Please refer to [End-to-End Testing in Kubernetes](e2e-tests.md).
+
+## Running your contribution in the Kubernetes CI
+Once you open a PR, [`prow`][prow-doc] runs pre-submit tests in CI.
+
+If you are not a [Kubernetes org member][member], another org member will need to run [`/ok-to-test`][ok-to-test] on your PR.
+
+Find out more about [other commands][prow-cmds] you can use to interact with prow through GitHub comments.
+
+### Troubleshooting a failure
+Click on `Details` and look at the [`gubernator`](gubernator.k8s.io/) output for the test.
```

Comment:
I would reword this as we are looking to move away from gubernator.  It also doesn't make much sense to a new contributor.  What am I going here to look for? Artifacts produced by the test and cluster under test, such as:
- test results
- more metadata about the tests (what versions of things were used? how long did they take?)
- output from tests that have failed
- build log showing the full test run
- logs from the cluster under test (k8s components such as kublet and apiserver, possibly other logs such as etcd and kernel)
- junit xml files
- coverage files
- etc.

Reply:
Thanks, removed the reference to gubernator and added specifics on what specifically they can find under `Details` that is useful to debugging. I haven't gone into a lot of detail as to _how_ to read through logs etc (I worry it might get too tiresome for folks to read & follow), any opinions on this?